extension/Hello/config/workflow.yaml

```diff
@@ -53,6 +53,20 @@ framework:
               settings:
                 failed: bye # transition name for filed result
                 successful: world # transition name for successful result
+        hello-interactive:
+          from: start
+          to: middle
+          metadata:
+            task: ~ # not neccessary to have a task for the interactive transition.
+            transitionResolver: # Resolver needs for resolve next transition
+              name: interactive
```

Comment:
please add a comment "By setting `interactive` transitionResolver name current transition will be recognized as interactive and appropriate actions will be done."